src/async_wrap.cc

```diff
@@ -850,7 +853,7 @@ MaybeLocal<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
   ProviderType provider = provider_type();
   async_context context { get_async_id(), get_trigger_async_id() };
   MaybeLocal<Value> ret = InternalMakeCallback(
-      env(), GetResource(), object(), cb, argc, argv, context);
+      env(), object(), object(), cb, argc, argv, context);
```

Comment:
So with this change, having the extra "resource" field on `InternalMakeCallback(...)` becomes unnecessary here. Just wondering if you have thoughts on if/how to expose the resource_symbol/owner_symbol stuff to userland for native modules to be able to handle resource re-use appropriately. It'd be nice if we could make #32930 irrelevant.

Reply:
@qard I would still think that #32930 is the right way forward for that – we kind of don’t have a choice with N-API, because the API & ABI are set in stone.

Reply:
And as far as `InternalMakeCallback()` is concerned … yes, that becomes unnecessary currently, but conceptually the JS receiver and the resource are independent, so it makes sense to pass them as independent arguments to me. If you feel strongly about it, I can merge them again, though.

Reply:
Nope, no strong feelings about it. Just wanted to know what your feelings are about it. Seemed to me a bit like the move toward `resource_symbol` and `owner_symbol` was going in a somewhat different direction from the changes in #32930 so it wasn't fully clear to me if it was still fully relevant or if you maybe had a plan on how to proceed without it.